apps/arena_load_test/lib/arena_load_test/socket_handler.ex

```diff
@@ -55,19 +55,22 @@ defmodule ArenaLoadTest.SocketHandler do
 
   # Private
   defp ws_url(player_id) do
-    host = SocketSupervisor.server_host()
     character = get_random_active_character()
     player_name = "Player_#{player_id}"
 
-    case System.get_env("SSL_ENABLED") do
-      "true" ->
-        "wss://#{host}/join/#{player_id}/#{character}/#{player_name}"
+    case System.get_env("SERVER_HOST") do
```

Comment:
Where is this set?

Reply:
woops. I forgot to set the input variable in the yaml file. It's done now!